src/mds/MDSRank.cc

```diff
@@ -2658,7 +2658,9 @@ void MDSRankDispatcher::handle_asok_command(
   } else if (command == "session evict" ||
 	     command == "client evict") {
     std::lock_guard l(mds_lock);
+    bool yes_i_really_mean_it = false;
     std::vector<std::string> filter_args;
+    cmd_getval(cmdmap, "yes-i-really-mean-it", yes_i_really_mean_it);
```

Comment:
`yes-i-really-mean-it` is for CLI, it would be `yes_i_really_mean_it` in cmdmap

Reply:
oops, I changed it by mistake. Will update